### ISSUE TITLE
It just makes one image and sits [t2v]

### ISSUE THREAD

**Issue by Ph0rk0z** [volotat/SD-CN-Animation#176]
Technically 2 if you use controlnet. No video is ever made, just one file is generated and I see it in the previews.

No errors, it just doesn't keep generating more. Deforum works fine, I have plenty of vram/compute.

**Comment by kzhjymug** [comment]
You didn't mention this, but are you using the latest A1111? See #173 , there is a known problem, something upstream changed that broke this project. There is no fix for now, so you would have to revert to some earlier commits to get t2v running again.

**Comment by Ph0rk0z** [comment]
That's true but all the issues I searched had other errors. Any idea which commits did it? Maybe there is a fix that can be made.
